lib/linter-plus.coffee

```diff
@@ -32,6 +33,9 @@ class Linter
 
     @subscriptions.add atom.workspace.observeTextEditors (editor) =>
       currentEditorLinter = new EditorLinter @, editor
+      currentEditorLinter.onLinting (linting) =>
+        @lintJobs += if linting then 1 else -1
```

Comment:
`if linting then @lintsJobs++ else @lintsJobs--`

But I think it's just preference of how to write it. (I find it clearer)